### DISCUSSION TITLE
Hovering the tray icon displays no information anymore? Windows 10

### DISCUSSION THREAD

**Discussion by joas47** [Tyrrrz/LightBulb#278] [General]
Hovering with the mouse over tray icon used give some information about the state of the program, like if it was disabled or enabled etc. It was a nice feature that seems to have been removed from version 2.4.8 onwards. I'm on Windows 10 and v2.4.9. Is it coming back?

**Comment by Tyrrrz** [comment]
Unfortunately, I had to revert to using the native tray icon/menu which doesn't support dynamic bindings. This was due to a move to Avalonia. I'll see if I can find a workaround.

**Comment by joas47** [comment]
This feature is gone again. I understand if it's not a priority but just thought you should know.

**Reply by Tyrrrz** [reply]
It's not fully gone, there's just a bug that causes the tooltip to miss the initial refresh. The current configuration values will still show up in the tooltip after they change. The bug will be fixed in the next release.
